src/date.py

```diff
@@ -44,7 +47,8 @@ def from_exif(self, exif, timestamp=None, user_regex=None, date_field=None):
         # sometimes exif data can return all zeros
         # check to see if valid date first
         # sometimes this returns an int
-        if datestr and isinstance(datestr, str) and not datestr.startswith('0000'):
+        if datestr and isinstance(datestr, str) and not \
```

Comment:
Can we keep this on one line?

Reply:
To keep it in one line is crossing the 79 characters per line requirement from PEP8.